server/cache.go

```diff
@@ -552,28 +561,25 @@ func (s *storeInfo) usedRatio() float64 {
 	return float64(s.stats.Stats.GetCapacity()-s.stats.Stats.GetAvailable()) / float64(s.stats.Stats.GetCapacity())
 }
 
-// usedRatioScore is the used capacity ratio of storage capacity, the score range is [0,100].
-func (s *storeInfo) usedRatioScore() int {
-	return int(s.usedRatio() * 100)
-}
-
 // clusterInfo is cluster cache info.
 type clusterInfo struct {
 	sync.RWMutex
 
-	meta        *metapb.Cluster
-	stores      map[uint64]*storeInfo
-	regions     *regionsInfo
-	clusterRoot string
+	meta          *metapb.Cluster
+	stores        map[uint64]*storeInfo
+	unknownStores map[uint64]struct{}
```

Comment:
`unknown store` means we have not received the store status heartbeat.
So we cannot select this store as a candidate.